src/app/pages/property-details/reviews/reviews.component.ts

```diff
@@ -0,0 +1,64 @@
+import { Subscription } from 'rxjs';
+import { Component, OnInit, Input } from '@angular/core';
+import { ClientReviewsService } from 'src/app/services/client-reviews/client-reviews.service';
+import { Router } from '@angular/router';
+import { NgxSpinnerService } from 'ngx-spinner';
+import { environment } from 'src/environments/environment.prod';
+
+
+
+@Component({
+  selector: 'app-reviews',
+  templateUrl: './reviews.component.html',
+  styleUrls: ['./reviews.component.scss']
+})
+export class ReviewsComponent implements OnInit {
+  @Input() clientId: number;
+  subscription: Subscription = new Subscription();
+  count: number;
+  results: any[];
+  url: string;
+  oneReview: any;
+  profileImage: string;
+
+  constructor(
+    private reviewsService: ClientReviewsService,
+    private router: Router,
+    public spinner: NgxSpinnerService,
+
+  ) {
+    this.profileImage = 'assets/img/people.png';
+   }
+
+  ngOnInit() {
+    this.fetchReviews();
+  }
+  fetchReviews() {
+    this.spinner.show();
+    const clientId = this.clientId;
+    this.url = `${environment.api_url}/auth/${clientId}/reviews/`;
+    this.subscription.add(
+      this.reviewsService.getReviews(this.url).subscribe(response => {
+        this.results = response.results;
+        this.oneReview = response.results[0];
+        this.count = response.count;
+        this.spinner.hide();
+      },
+       _ => {
+          this.spinner.hide();
+      })
+    );
+  }
+  getMoreReviews(client: number) {
+    this.spinner.show();
+    this.router.navigate(['auth', this.oneReview.client, 'reviews']);
+    this.spinner.hide();
+  }
+
+  ngOnDestroy(): void {
```

Comment:
The Lifecycle interface `OnDestroy` should be implemented on line 15 for the `ngOnDestroy` method to work correctly. I have left s suggestion for it on line 15.